package login.proto;


enum ClusterStatus {
 STATUS_ONLINE = 1;
 STATUS_OFFLINE = 2;
 STATUS_INVITE_REQUIRED = 3;
 STATUS_INSUFFICIENT_PRIVILEGES = 4;
 STATUS_BANNED = 5;
}
